tests/serverpod_test_server/test/database_operations/generated_operations/batch_interactions_generated_test.dart

```diff
@@ -0,0 +1,615 @@
+import 'dart:typed_data';
+
+import 'package:serverpod_test_client/serverpod_test_client.dart';
+import 'package:test/test.dart';
+
+import '../../config.dart';
+
+void main() {
+  var client = Client(serverUrl);
+
+  tearDown(() async => await client.databaseBatchGenerated.deleteAll());
```

Comment:
Question: What is the difference between this test class and `tests/serverpod_test_server/test/database_operations/generated_operations/batch_interactions_test.dart`?

Reply:
Seems the files are identical but uses a different interface? Most comments there seems like they apply here as well.

Reply:
yepp they are exactly the same but call different endpoints. The endpoints differ only by one calling `session.db.<insert|update|delete|...>` and the other calling `DataType.db.<insert|update|delete|..>`